src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/BOM/MaterialForm/MaterialFormView.tsx

```diff
@@ -184,7 +185,7 @@ const MaterialFormView: React.FC<MaterialFormViewProps> = ({
               <ReactHookTextField
                 name="quantity"
                 control={control}
-                errorMessage={errors.quantity}
+                errorMessage={errors.quantity?.root} // I just wrote root here for the ts test to pass but not too sure what should go here because I was getting a type error
```

Comment:
it's because you set quantity to a Decimal, decimal might not be compatible with React-hook-forms. We might wanna keep it as a number (bc/ I believe that still keeps info about the decimal places) while it's in the form, then convert it to a Decimal to send to the backend